index.ts

```diff
@@ -1,33 +1,26 @@
-import { AuditsClient } from './src/audits';
 import { FronteggAuthenticator } from './src/authenticator';
-import { IdentityClient } from './src/identity';
-import { contextResolver, withAuthentication, contextResolverWithPermissions } from './src/identity';
-import { ContextHolder, frontegg, fronteggNextJs, IFronteggOptions } from './src/middleware';
-import { NotificationsClient } from './src/notifications';
-import { FronteggPermissions } from './src/permissions';
-import { RbacMiddleware } from './src/rbac';
-import { SsoClient } from './src/sso';
-import { TenantsClient } from './src/tenants';
-import { AuthzClient } from './src/authz';
-import { HttpClient } from './src/http';
+import { ContextHolder } from './src/components/context';
+import { withAuthentication } from './src/middlewares';
+import {
+  AuditsClient,
+  NotificationsClient,
+  TenantsClient,
+  SsoClient,
+  IdentityClient,
+  AuthzClient,
+  HttpClient,
+} from './src/clients';
 
-export * from './src/events';
+export * from './src/clients/events';
 
 export {
   AuditsClient,
   ContextHolder,
   NotificationsClient,
   TenantsClient,
-  frontegg,
-  fronteggNextJs,
```

Comment:
check with @aviadmizrachi or @davidantoon

Reply:
David said they transferred the fronteggNextJs code to their own repository, so this can be deleted